Sources/ViewInspector/SwiftUI/CustomView.swift

```diff
@@ -2,15 +2,15 @@ import SwiftUI
 
 @available(iOS 13.0, macOS 10.15, tvOS 13.0, *)
 public protocol CustomViewType {
-    associatedtype T: InspectableProtocol
```

Comment:
Now that environment injection is decoupled from `InspectableProtocol`, we no longer need this requirement.